### ISSUE TITLE
Elements in list view missing label attribute, leading to unnecessary warnings

### ISSUE THREAD

**Issue by gabriel-bider-nltg** [umbraco/Umbraco-CMS#21923]
### Which Umbraco version are you using?

17.2.0

### Bug summary

In the Collection/List View in Content, the checkboxes and buttons seem to be missing a `label` attribute which leads to warnings in the Console.

### Steps to reproduce

Checking the checkboxes reveals buttons below the table which also don't seem to have any `label` attribute, leading to similar warnings.

<img width="970" height="351" alt="Image" src="https://github.com/user-attachments/assets/6e97e41a-b004-400d-8b86-66ad167d5ff1" />

<img width="1116" height="144" alt="Image" src="https://github.com/user-attachments/assets/7596c162-2bf0-4ffc-96d2-9c060f954722" />

<img width="590" height="208" alt="Image" src="https://github.com/user-attachments/assets/9b25933e-b9f5-46ff-a60f-55e8a82781e9" />

### Expected result / actual result

No warnings printed in the Console

**Comment by andreaslborg** [comment]
Hi @leekelleher and @gabriel-bider-nltg,

I have fixed the warnings in the selection-info container. However, when adding a label to the checkboxes in the table, the label is rendered visually as well, which does not look correct:
<img width="878" height="490" alt="Image" src="https://github.com/user-attachments/assets/a8484bd2-f3d4-4879-999d-c6eb394db7a7" />

There is a possible hotfix where we simply add a space as the label, but this does not look great either:
<img width="1376" height="634" alt="Image" src="https://github.com/user-attachments/assets/66dfee2c-0b5c-4736-aeb5-b02f8d8c384b" />

Would it be possible to support checkboxes without triggering the warning when an accessible name is already provided, without rendering visible label text?

I think the best solution would be to update the warning check to also consider `aria-label`.
If that is not possible, a workaround would be to use a Zero Width Space (ZWSP), e.g. `"\u200B"`, as the label to avoid the warning.

**Comment by andreaslborg** [comment]
Hi again @leekelleher and @gabriel-bider-nltg,
I have made a PR #21964 for this, as it doesn't seem possible to update the UUI check to consider `aria-label`.

**Comment by ghost** [comment]
This sounds great, thanks! @andreaslborg

**Comment by andreaslborg** [comment]
I agree with the review @leekelleher has made in my PR about the checkbox workaround. I've therefore made a bug report in the [Umbraco.UI repo](https://github.com/umbraco/Umbraco.UI/issues/1332). @gabriel-bider-nltg if you have any additions to the PR there, please let me know. 😊

**Comment by leekelleher** [comment]
PR #21964, (addresses the entity bulk action button label), has been merged in, for v17.3.0.

We'll follow up with the UUI checkbox label soon.

**Comment by andreaslborg** [comment]
@leekelleher I have been told:
> This has been addressed via https://github.com/umbraco/Umbraco.UI/pull/1333 for the next V1 release. It will be further addressed in the V2 release by moving to ElementInternals to avoid forwarding these attributes to the inner element entirely.
So I believe you can close this now. 😊

**Comment by leekelleher** [comment]
@andreaslborg Thanks for following up on this! I'll mark this as closed for v17.3.0.
